client/src/components/pages/addFlat/AddFlat.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import Input from '../../../common/input/Input.jsx';
+import Button from '../../../common/button/Button.jsx';
+import ImageUploader from 'react-images-upload';
+import './addFlat.scss';
+
+class AddFlat extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      street: '',
+      house_number: '',
+      corps_number: '',
+      flat_number: '',
+      flore_number: '',
+      square_meters_number: '',
+      price_per_month: '',
+      discription: '',
+      pictures: []
+    };
+    this.onDrop = this.onDrop.bind(this);
+  }
+
+  togglePassword = () => {
+    if (this.state.isTypeVisible === 'password') {
+      this.setState({ isTypeVisible: 'text' });
+    } else this.setState({ isTypeVisible: 'password' });
+  };
+
+  onChange = e => this.setState({ [e.target.name]: e.target.value });
```

Comment:
pls use destructuring

```suggestion
  onChange = ({ target }) => this.setState({ [target.name]: target.value });
```